### DISCUSSION TITLE
Language constraint

### DISCUSSION THREAD

**Discussion by ahnineamine** [geeksblabla/blanat#11] [Q&A] [answered]
I know it's a bit redundant to ask since you already set the constraint, but would it be possible to develop the solution using a language other than the ones mentioned? Kotlin, for example?
Thanks guys!

**Comment by aboullaite** [comment, marked as answer]
We wanted to but unfortunately we ran out of time. The runner to evaluate the solutions only supports those languages for this round. We will add more language support in future editions 🙏🏼
